source/grid/tria.cc

```diff
@@ -9823,6 +9879,19 @@ Triangulation<dim, spacedim>::create_triangulation(
       throw;
     }
 
+  if (true)
```

Comment:
Here I would like to use `all_reference_cell_types_are_hyper_cube()` from PR #11140.